content/docs/developing-operators/tasks.md

```diff
@@ -47,6 +47,27 @@ tasks:
 
 **Note:** deleting non-existing resources is always successful. As of version 0.9.0, KUDO will not wait for the resource to be actually removed and will finish the task when the API server accepts the deletion request. So in case of Pods, Kubernetes imposes a default graceful termination period of 30 seconds, however, a delete-task will be done before that. KUDO 0.9.0 will not wait for [resource finalizers](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/) should they exist.
 
+
+## Toggle-Task
```

Comment:
Could you please mention Toggle-Tasks [in this paragraph](https://github.com/kudobuilder/kudo.dev/pull/209/files#diff-f9e3827927df0720222e93b2a95594b3R5)?

Reply:
added to the overview